.github/workflows/recent-github-activity.yml

```diff
@@ -0,0 +1,17 @@
+name: Recent Github Activity Update README
+
+on:
+  schedule:
+    - cron: "*/30 * * * *"
```

Comment:
note this is a cron action for every 30 minutes. Reconsider late if you want to update it to `0 0 1/1 * *` to have it happening everyday at 12:00 AM O'clock (midnight)